.github/ISSUE_TEMPLATE.md

```diff
@@ -1,11 +1,10 @@
 ## Context
 
-**Issue type** (bug report or feature request): 
-**Operating system** (e.g. Fedora 24, Mac OS 10.11, Windows 10): 
-**Platform** (e.g. 64-bit x86, 32-bit ARM): 
-**OpenSlide Python version** (`openslide.__version__`): 
-**OpenSlide version** (`openslide.__library_version__`): 
-**Slide format** (e.g. SVS, NDPI, MRXS): 
+**Issue type** (bug report or feature request):
+**Operating system** (e.g. Fedora 24, Mac OS 10.11, Windows 10):
+**Platform** (e.g. 64-bit x86, 32-bit ARM):
+**OpenSlide Python version** (`openslide.__version__`):
+**OpenSlide version** (`openslide.__library_version__`):
+**Slide format** (e.g. SVS, NDPI, MRXS):
```

Comment:
This is a form that we expect users to fill out, and the trailing space is there for their convenience.  Can we ignorelist this file?

Reply:
I assume that can be ignored, maybe we should just ignore everything in .github?

Reply:
Could do.  It might make sense to continue checking workflows, though.